libs/net/dal/Migrations/1.0.81/Up/PostUp/00-Report_Template.sql

```diff
@@ -0,0 +1,203 @@
+                       if (section.Value.Settings.ShowFullStory)
+                       {
+                         <li><a href="#item-@content.Id">@headline</a></li>
+                       }
+                       else
+                       {
+                         <li><a href="@($"{@ViewContentUrl}{content.Id}")" target="_blank">@headline</a></li>
+                       }
+                    }
+                  </ul>
+                }
+                </ul>
+              }
+              else
+              {
+                 <ul style="margin:0; margin-left: 25px; padding:0;">
+                 @foreach (var content in sectionContent)
+                 {
+                   var headline = $"{(Settings.Headline.ShowSentiment ? content.GetSentimentIcon("{0} - ") : "")}{content.Headline}{(Settings.Headline.ShowShortName && !String.IsNullOrEmpty(content.Source?.ShortName) ? $" - {content.Source?.ShortName}" : (Settings.Headline.ShowSource ? $" - {content.OtherSource}": ""))}{(Settings.Headline.ShowPublishedOn ? $" - {content.PublishedOn?.AddHours(utcOffset):dd-MMM-yyyy}" : "")}";
+                    @if (section.Value.Settings.ShowFullStory)
+                    {
+                      <li><a href="#item-@content.Id">@headline</a></li>
+                    }
+                    else
+                    {
+                      <li><a href="@($"{@ViewContentUrl}{content.Id}")" target="_blank">@headline</a></li>
+                    }
+                }
+                </ul>
+              }
+          }
+
+          @* Full Stories *@
+          @if (section.Value.Settings.ShowFullStory)
+          {
+            for (var i = 0; i < sectionContent.Length; i++)
+            {
+              var content = sectionContent[i];
+              var hasPrev = i > 0;
+              var prev = hasPrev ? sectionContent[i - 1].Id : 0;
+              var hasNext = (i + 1) < sectionContent.Length;
+              var next = hasNext ? sectionContent[i + 1].Id : 0;
+              <div>
+                <div id="item-@content.Id" style="display: flex; align-items: center;">
+                  <a name="item-@content.Id"></a>
+                </div>
+                <div>
+                  <a href="#top">top</a>
+                  @if (i > 0)
+                  {
+                    <a href="#item-@prev">previous</a>
+                  }
+                  @if (i < sectionContent.Length -1)
+                  {
+                    <a href="#item-@next">next</a>
+                  }
+                </div>
+                <h3>@content.Headline</h3>
+                <div>@content.Source?.Name</div>
+                <div>@content.PublishedOn?.AddHours(utcOffset).ToString("dddd, MMMM d, yyyy")</div>
+                @if (!string.IsNullOrWhiteSpace(content.Page))
+                {
+                    <div>Page @content.Page</div>
+                }
+                @if (!string.IsNullOrWhiteSpace(content.Byline) && Settings.Headline.ShowByline)
```

Comment:
This is the only line changed in the template since my change.  The key point is this new check
`&& Settings.Headline.ShowByline`

Reply:
Exactly ! tks !